clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-artifact-EC2-TGMappings-multipleLBsAndContainers.json

```diff
@@ -0,0 +1,53 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-2": [
+      "us-west-2a",
+      "us-west-2c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "integArtifactEC2TgMappingskWithMultipleLBsAndContainers-cluster",
+  "launchType": "EC2",
+  "networkMode": "bridge",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integArtifactsEC2TgMappingsStackWithMultipleLBsAndContainers",
+  "freeFormDetails" : "detailTest",
+  "targetGroupMappings": [
+    {
+      "containerName": "application1",
+      "containerPort": 80,
+      "targetGroup": "integArtifactEC2TgMappings-targetGroupForPort80"
+    },
+    {
+      "containerName": "application2",
+      "containerPort": 84,
+      "targetGroup": "integArtifactEC2TgMappings-targetGroupForPort84"
+    }
+  ],
+  "useTaskDefinitionArtifact" : true,
+  "taskDefinitionArtifactAccount"  :  "my-github",
+  "resolvedTaskDefinitionArtifact": {
+    "account": "ecs-account",
+    "type": "ecs",
+    "customKind" :  true,
+    "name" : "applications",
+    "location" : "us-west-2",
+    "reference" : "refernce",
+    "metadata" : null,
+    "artifactAccount" : "my-github",
+    "provenance" : "prov",
+    "uuid" : "uid-123"
+  },
+  "containerToImageMap": {}
```

Comment:
this almost always contains one or more docker images the user wants to insert into the task def artifact. Can we update this test to supply this map and assert that the correct container image is inserted into each container definition of the `registerTaskDefinition` call?

Reply:
make sense, I have added the container to image map and asserting on the image value that container holds.